dingo/gw/transforms/detector_transforms.py

```diff
@@ -340,6 +356,7 @@ def __init__(
                     self.data_domain.f_max,
                     num_calibration_nodes,
                     ifo.name,
+                    correction_type=correction_type,
```

Comment:
So `correction_type` just gets forwarded to the relevant Bilby object? Are the terms "data" and "template" taken from there?

Do we need to specify a minimum Bilby version for this to work?

Reply:
I don't think we need to remove support for older versions of Bilby just for this though. Many users won't even use calibration marginalization so this won't affect them.